omero/sysadmins/cli/LightAdmins.txt

```diff
@@ -0,0 +1,61 @@
+Adjusting administrator restrictions
+====================================
+
+OMERO 5.4 introduced the concept of a :doc:`restricted administrator
+<../admins-with-restricted-privileges>`. The meaning and representation
+of the server's underlying permissions restrictions is described in
+:doc:`developer documentation</developers/Server/LightAdmins>`.
+OMERO.web offers easy management of restrictions and is recommended for
+setting up restricted administrators via its :help:`Admin tab
+<facility-manager#lightadmin>`.
+
+OMERO.cli does not offer easy management of restrictions because support
+is yet to be added. In the meantime it can already manipulate
+administrator restrictions in the awkward manner described hereunder.
+
+.. warning::
+
+  OMERO.web provides a simplified view of the available restrictions:
+  *one* checkbox offered in the web interface may represent *multiple*
+  underlying restrictions. The recommended OMERO.web management
```

Comment:
But in web the checkboxes must be viewed as privileges. I think this shoiuld ba mentioned here. 
``one checkbox in web == multiple privileges given back to the admin from whom all the restrictions were applied to start with as a default``